module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -316,6 +332,9 @@ protected function getLibGuidesDatabases()
             $nameToDatabase = [];
             foreach ($databases as $database) {
                 $nameToDatabase[$database->name] = (array)$database;
+                if ($this->useLibGuidesAlternateNames && ($database->alt_names ?? false)) {
+                    $nameToDatabase[$database->alt_names] = (array)$database;
```

Comment:
I notice that the API field is plural (alt_names) but we're treating it as a single flat string. What happens when there are multiple values here? Is there a standard delimiter, or is this just a free-text field that's used inconsistently?

I realize that we're doing a str_contains search against this value, so it should work even if the field contains multiple names... but if there is a standard and reliable delimiter, it feels like it might be worth splitting it up to make our index a little more accurate -- that might eliminate a certain rare subset of false positive hits and/or provide an opportunity for smarter relevance ranking logic in the future.

Reply:
Yeah, unfortunately is a plan single-line string field.  I have seen a value like "Used to be called XX" which is probably not what was intended and of course would cause problems if someone actually searched for "used to" for some ^$E@#%! reason.  So I went with a naive algorithm.  In any case that's what LibGuides would be doing itself to search against that field.

Reply:
Makes sense! As I noted, it's then probably worth adding a comment like:

```
// The alt_names field is single-valued free text; we'll do the best we can with it
```

:-)

Reply:
Agreed, sorry I missed the comment suggestion the first time around.